src/polyfem/io/OutData.cpp

```diff
@@ -48,6 +49,74 @@ extern "C" size_t getPeakRSS();
 
 namespace polyfem::io
 {
+	namespace
+	{
+		void compute_traction_forces(const State &state, const Eigen::MatrixXd &solution, Eigen::MatrixXd &traction_forces, bool skip_dirichlet = true)
```

Comment:
this is the integrated tf, right? is so they maybe change the name of the func?

Reply:
This is the traction force, which is the integrated pressure. The "Stress * normal" is the pressure.